apps/web/src/components/transactions/TxDetails/TxNote.tsx

```diff
@@ -0,0 +1,59 @@
+import { Tooltip, Typography } from '@mui/material'
+import type { TransactionDetails } from '@safe-global/safe-gateway-typescript-sdk'
+import useAsync from '@/hooks/useAsync'
+import { useCurrentChain } from '@/hooks/useChains'
+import { isMultisigDetailedExecutionInfo } from '@/utils/transaction-guards'
+import InfoIcon from '@/public/images/notifications/info.svg'
+
+function useTxNote(txDetails: TransactionDetails | undefined): string | undefined {
+  const currentChain = useCurrentChain()
+  const txService = currentChain?.transactionService
+
+  let safeTxHash = ''
+  if (txDetails && isMultisigDetailedExecutionInfo(txDetails.detailedExecutionInfo)) {
+    safeTxHash = txDetails.detailedExecutionInfo?.safeTxHash
+  }
+
+  const [data] = useAsync(() => {
+    if (!safeTxHash || !txService) return
+    return fetch(`${txService}/api/v1/multisig-transactions/${safeTxHash}`).then((res) => res.json())
+  }, [safeTxHash, txService])
```

Comment:
This will be replaced by https://github.com/safe-global/safe-client-gateway/pull/2223